dashboard/app/controllers/sections_controller.rb

```diff
@@ -5,15 +5,15 @@ class SectionsController < ApplicationController
   authorize_resource :section, only: [:new]
 
   def new
-    return head :forbidden unless current_user.admin
+    return head :forbidden unless current_user
```

Comment:
I'm not sure you need these now actually. `authorize_resource` _should_ take care of this I think

Reply:
I wasn't sure! Okay awesome will delete

Reply:
Worth double checking really quickly but I think that should be true